gateserver/messagehandler.py

```diff
@@ -0,0 +1,28 @@
+import socketserver
+import psycopg2
+
+class MessageHandler(socketserver.BaseRequestHandler):
+    """Handles a message from the controller.
+
+    Behaves according to
+    https://github.com/fmfi-svt/gate/wiki/Controller-%E2%86%94-Server-Protocol .
+    """
+
+    _db_conn = None
+
+    @classmethod
+    def set_db(cls, db_conf):
+        """Connects to the database. All instances will share the connection."""
+        cls._db_conn   = psycopg2.connect(db_conf)
+
+    def handle(self):
+        data, socket = self.request
+        print("Received data from {}:".format(self.client_address))
+        print(data)
+
+        cur = self._db_conn.cursor()
+        cur.execute("SELECT * FROM controllers;")
```

Comment:
Mozno je zvykom bez bodkociarky.
Chystame sa podporovat aj ine backendy (povedzme sqlite pri developerskom testovani)? Budeme niekedy potrebovat nejake zlozitejsie queries? Ak tak by sme mohli zvazit sqlalchemy. Ak nie, tak nevermind a som za nechat to takto.